Index: frontend/src/App.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import SearchPage from \"./SearchPage\";\r\nimport \"./App.css\";\r\n\r\nfunction App() {\r\n  return <SearchPage />;\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/frontend/src/App.jsx b/frontend/src/App.jsx
--- a/frontend/src/App.jsx	(revision 1479aadd3b73e6fc58c268bf6bf7cc8c22ec61c0)
+++ b/frontend/src/App.jsx	(date 1782250147211)
@@ -1,8 +1,21 @@
-import SearchPage from "./SearchPage";
-import "./App.css";
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import MainPage from './mainPage'; // ახალი მთავარი გვერდი
+import SearchPage from './searchPage'; // დიდი სერჩის გვერდი
+import './App.css';
 
 function App() {
-  return <SearchPage />;
+  return (
+      <Router>
+        <Routes>
+          {/* საიტზე შესვლისას (localhost:5173/) გაიხსნება MainPage */}
+          <Route path="/" element={<MainPage />} />
+
+          {/* ძებნაზე დაჭერისას გადავა /search მისამართზე და გამოჩნდება SearchPage */}
+          <Route path="/search" element={<SearchPage />} />
+        </Routes>
+      </Router>
+  );
 }
 
-export default App;
+export default App;
\ No newline at end of file
